pyHS100/pyHS100.py

```diff
@@ -285,6 +268,156 @@ def identify(self):
 
         return alias, model, features
 
+    @property
+    def alias(self):
+        """
+        Get current device alias (name)
+        :return:
+        """
+        return self._alias
+
+    @alias.setter
+    def alias(self, alias):
+        """
+        Sets the device name aka alias.
+        :param alias: New alias (name)
+        """
+
+        self._query_helper("system", "set_dev_alias", {"alias": alias})
+
+    @property
+    def led(self):
+        """
+        Returns the state of the led.
+        :return:
+        """
+        return {"led": 1 - self.sys_info["led_off"]}
+
+    @led.setter
+    def led(self, state):
+        """
+        Sets the state of the led (night mode)
+        :param state: 1 to set led on, 0 to set led off
+        """
+
```

Comment:
Done. Would it be good idea also to return empty :returns ? My argument being, when the description already tells what it does, e.g. in case of led().